pkg/container/types/date.go

```diff
@@ -43,6 +43,33 @@ const (
 	Saturday
 )
 
+// String returns the English name of the day ("Sunday", "Monday", ...).
+func (d Weekday) String() string {
+	if Sunday <= d && d <= Saturday {
+		return longDayNames[d]
+	}
+	buf := make([]byte, 20)
+	n := fmtInt(buf, uint64(d))
+	return "%!Weekday(" + string(buf[n:]) + ")"
+}
+
+// fmtInt formats v into the tail of buf.
+// It returns the index where the output begins.
```

Comment:
Why this?   itoa is not good enough?

Reply:
These judgment logics are no longer needed. I'll delete them